extension.meta.xml

```diff
@@ -15,7 +15,7 @@
 	<media>
 	</media>
 	<releases>
-		<release version="1.0" date="2013-02-02" min="2.3">
```

Comment:
please *add* a new `<release...>` element and only adjust the `max` attribute on the old releases.